R/RLearner_regr_gbm.R

```diff
@@ -13,14 +13,16 @@ makeRLearner.regr.gbm = function() {
       makeNumericLearnerParam(id = "shrinkage", default = 0.001, lower = 0),
       makeNumericLearnerParam(id = "bag.fraction", default = 0.5, lower = 0, upper = 1),
       makeNumericLearnerParam(id = "train.fraction", default = 1, lower = 0, upper = 1),
+      makeNumericLearnerParam(id = "alpha", default = 0.5, lower = 0, upper = 1),
```

Comment:
Please make this param dependent on `quantile`. No need to add a statement in the learner note.